fdbclient/ClientKnobs.cpp

```diff
@@ -180,6 +180,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 	init( RESTORE_DISPATCH_ADDTASK_SIZE,           150 );
 	init( RESTORE_DISPATCH_BATCH_SIZE,           30000 ); if( randomize && BUGGIFY ) RESTORE_DISPATCH_BATCH_SIZE = 20;
 	init( RESTORE_WRITE_TX_SIZE,            256 * 1024 );
+	init( RESTORE_LOAD_KEY_VERSION_MAP_STEP_SIZE, 10000);
```

Comment:
Do we want to buggify this knob?

Reply:
My guess is no since we add this knob to tune performance and it should not affect correctness